pkg/generate/app/cmd/newapp.go

```diff
@@ -218,6 +218,11 @@ func (c *AppConfig) SetOpenShiftClient(osclient client.Interface, originNamespac
 		ClientMapper: c.clientMapper,
 		Namespace:    originNamespace,
 	}
+	c.dockerSearcher = app.ImageImportSearcher{
```

Comment:
SetDockerClient is always invoked before SetOpenShiftClient unless you can point me where that's not the case.